platform/server/src/main/webapp/static/js/bootstrap-page.js

```diff
@@ -9,13 +9,16 @@ function verifyDbConnection() {
     var warnings = $('#verify-alert');
     var infoSql = $('#verifySql-info');
     var errors = $('#verify-error');
+    var bootstrapErrors = $('#bootstrapErrors');
```

Comment:
do we need to be clearing the html for this div? From what I understand these errors only come in after doing a post to the server anyway